### ISSUE TITLE
Kb/s to Mb/s or MB/s conversion

### ISSUE THREAD

**Issue by tx7** [firehol/netdata#616]
Hi,

What's the easiest way to convert Kbs to MBs for IPv4, Read/Write disk for the graphs?
Or at least Mbs.
Will this implement in a future release?
..........................................................................
Second Note:

I was doing some RAM usage comparison between Netdata and Webmin:

RAM usage comparison:
Webmin: use 20%, free 80%
OpenMediaVault: use 20%, free 80%
Netdata: use 25.11%   , free 74.89 %

The average ram usage difference is 5% higher from Netdata.
Why?

Thanks.

**Comment by ktsaou** [comment]
You can change the divider and the units label for any chart in netdata.conf. Just search for the chart name and you see the options.

Regarding RAM, netdata reports this:

![image](https://cloud.githubusercontent.com/assets/2662304/16426487/e4910da4-3d71-11e6-9712-03faacedd200.png)

At the top gauge chart, it reports as **free**: `cached` and `free`. `used` and `buffers` are considered **used**.

The information is read from `/proc/meminfo`.

I don't know what the other tools are doing, but generally the **available ram** is subject to interpretation. A few tools report `available = total - used`, but this is misleading since you cannot zero `cached` and `buffers`. I thought that `available = free + cached` to be closer to what you can get before swapping kicks in.

**Comment by tx7** [comment]
So, if I need to change Kbs to Mbs, I need to multiply another 1000 ?

**Comment by ktsaou** [comment]
Yes, to convert kilobits/s to megabits/s, whatever there is on the `divider` line, multiply it by 1000.
Change also the `units` to reflect this change.

**Comment by tx7** [comment]
I think you meant `dim received divisor`, and `dim sent divisor`.

**Comment by ktsaou** [comment]
Yes, these are the ones.
Yes, you have to uncomment the lines.

**Comment by tx7** [comment]
I'm able to change the network interfaces graph to megabits/s, but I can't find where the units for the IPv4 of the system overview.

**Comment by ktsaou** [comment]
This is my `system.ipv4`:

```
[system.ipv4]
	# history = 172800
	# enabled = yes
	# cache directory = /var/cache/netdata/system.ipv4
	# chart type = area
	# type = system
	# family = network
	# context = system.ipv4
	# units = kilobits/s
	# priority = 500
	# gap when lost iterations above = 1
	# name = system.ipv4
	# title = IPv4 Bandwidth (system.ipv4)
	# dim received name = received
	# dim received algorithm = incremental
	# dim received multiplier = 8
	# dim received divisor = 1024
	# dim sent name = sent
	# dim sent algorithm = incremental
	# dim sent multiplier = -8
	# dim sent divisor = 1024
```

As you can see, there is a `units` entry.

**Comment by ktsaou** [comment]
Oh, I see, you mean the gauge chart. Let me check.

**Comment by ktsaou** [comment]
You are right. They are hardcoded in `index.html`. I did that to make them shorter... The easy fix is to remove the units from `index.html` and they will inherit whatever you write in `netdata.conf`.

I'll create a pull request tomorrow to fix them.

**Comment by tx7** [comment]
Yes, the gauge chart.
 No wonder I couldn't find them b/c it's in index.html.

Thanks for your help.

**Comment by ktsaou** [comment]
Oops! I forgot you.
I am going to do it now.

**Comment by ktsaou** [comment]
ok. Committed the fix. It will be merged with #687 
Sorry for the delay...

**Comment by ktsaou** [comment]
I have moved all the unit conversion requests to #2745 
So, let's continue this discussion there.
